### DISCUSSION TITLE
Some peruvian channels are not shown

### DISCUSSION THREAD

**Discussion by josepedro770** [iptv-org/community#997] [Q&A]
### Describe your issue

Hello. I have noticed that in the list corresponding to Peru the following channels are not displayed or have been removed:

- Best Cable Mas Cumbia
- Best Cable Sports
- Hatun TV
- TV Perú
- TV Perú Noticias

Please put them back.

Thanks since now.

Greetings.
[Abrir en Google T](https://translate.google.com/?gs_lcrp=EgZjaHJvbWUqDggAEEUYJxg7GIAEGIoFMg4IABBFGCcYOxiABBiKBTIMCAEQRRg5GLEDGIAEMg0IAhAAGIMBGLEDGIAEMg0IAxAAGIMBGLEDGIAEMg0IBBAAGIMBGLEDGIAEMg0IBRAAGIMBGLEDGIAEMgoIBhAAGLEDGIAEMg0IBxAAGIMBGLEDGIAEMgcICBAAGIAEMgcICRAAGIAEqAIAsAIA&um=1&ie=UTF-8&hl=es&client=tw-ob#es/en/Hola.%20He%20notado%20que%20en%20la%20lista%20corresponidnte%20a%20Peru%20no%20se%20visualizan%20o%20han%20sido%20quitados%20los%20siguentes%20canales:%0A%0APor%20favor,%20coloquenlos%20nuevamente.%0A%0AGracoas%20desde%20ya.%0A%0ASaludos.)

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

**Comment by BellezaEmporium** [comment]
EN : If the channels do not appear in the list, is that they're likely to be either unavailable or non-existent. It may happen if we see that the link we used to have do not work today.

I'll take a check on your channel request after IDing a few LATAM channels.

ES : Si los canales no aparecen en la lista, es que probablemente no estén disponibles o no existan. Puede ocurrir si vemos que el enlace que teníamos antes no funciona hoy.

Revisaré tu solicitacion de canal despues de identificar algunos canales de LATAM.

**Reply by josepedro770** [reply]
I have provided new streaming links that worked for the mentioned channels and replaced the broken links. But they have not replaced them, but have directly deleted them without any explanation and without even fixing the links that I have provided. Below I send the links so that they can be fixed and placed back in the corresponding list:

- Best Cable Mas Cumbia: https://mi.godo.net.pe/best-MasCum-0102210011/tracks-v1a1/mono.m3u8?remote=190.64.92.146&token=4f0d78cc628e19bbd5d5628e0c94b2deb6733b1a-5e40e1f478c56c9940f48fd405a21558-1702933407-1702922607

- Hatun TV: https://mi.godo.net.pe/Best00Hatum-0032101002/tracks-v1a1/mono.m3u8?remote=190.64.92.146&token=43f579418469b06d9ecfa8b5b6d86b052f9401af-d3735fb03e62aae2da5132ee4238c778-1702933460-1702922660

- TV Perú: https://cdnhd.iblups.com/hls/777b4d4cc0984575a7d14f6ee57dbcaf.m3u8 

- TV Perú Noticias: https://cdnhd.iblups.com/hls/902c1a0395264f269f1160efa00660e4.m3u8 

I appreciate you being able to place and replace these signs.
Greetings

**Reply by PopeyeTheSai10r** [reply]
Only one link worked for.
The rest are either not working or geo-locked.

**Reply by freearhey** [reply]
TV Perú and TV Perú Noticias were added to the playlist in the last update: https://github.com/iptv-org/iptv/commit/b7ea8d970fdda28a186c0a6abead5b92ecef6735

Best Cable Mas Cumbia and Hatun TV is still under review: https://github.com/iptv-org/iptv/issues/15361, https://github.com/iptv-org/iptv/issues/15360

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
